### PR TITLE
Another try to run travis tests with docker

### PR DESCRIPTION
**Pull request by Ladsgroup**
Bug: T183214

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,43 +1,24 @@
...
+env:
+  - DOCKER_COMPOSE_VERSION=1.17.1
+services:
+  - docker
 before_install:
-  - npm install -g eslint
-  - sudo sed -e 's/^bind.*/bind 127.0.0.1/' /etc/redis/redis.conf > redis.conf
-  - sudo mv redis.conf /etc/redis
-  - sudo service redis-server start
-  - echo PING | nc localhost 6379
```
Comment (halfak) [inline .travis.yml]: What's going on here?  Looks like this needs a comment.

File: .travis.yml [hunk 2]
```diff
@@ -1,43 +1,24 @@
...
-  - flake8 .
-  - eslint .
-  - ./utility applications.wsgi --config ./config/celery-test/ &
-  - ./utility applications.celery --config ./config/celery-test/ &
-  - sleep 5
-  - ./utility test_api http://localhost:8080 --debug
-  - py.test --cov=ores -vv
+  - docker-compose up -d
+  - docker exec ores_ores-worker_1 flake8 /ores
+#  - eslint .
```
Comment (halfak) [inline .travis.yml]: How come we're not running this?

File: .travis.yml (hunk 1 above)
Comment (Ladsgroup) [reply .travis.yml]: All of this is being handled in the docker-compose. Docker compose brings up the redis and all of the needed dependencies.

File: .travis.yml (hunk 2 above)
Comment (Ladsgroup) [reply .travis.yml]: That's the next thing in my list to tackle. It's not hard though. I just need to do it.

Comment (Ladsgroup) [reply .travis.yml]: Also I want to mention this is linting for js/css. We don't have much of it in ores. I will fix it though.

File: .travis.yml (hunk 1 above)
Comment (halfak) [reply .travis.yml]: I think it's showing the wrong side of the diff.  See, the code you added removes docker-compose and re-downloads it.  Why?

Comment (Ladsgroup) [reply .travis.yml]: aha, docker compose is not installed by default, it needs to be installed: https://docs.travis-ci.com/user/docker/#using-docker-compose

Comment (Ladsgroup) [reply .travis.yml]: I removed that part completely, it's not needed.

**Comment by halfak** [comment]
OK.  It looks like we need a follow-up task for eslint, but otherwise, this looks good.

**Pull request merged**